crates/network-scanner-net/examples/basic_ping.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(unused_crate_dependencies)]
```

Comment:
Maybe let's create a separate issue to address removing these kinds of lints later?

Reply:
Agreed, my intention is to send another PR to rework the workspace. I’ll follow up on this soonish.